app/views/allowances/AllowancesStartView.scala.html

```diff
@@ -41,9 +41,17 @@ <h1 class="govuk-heading-l">@messages("allowancesStart.heading")</h1>
         }
         </ul>
 
+        @if(cashOrAccruals) {
         @govukButton(
             ButtonViewModel(messages("site.continue"))
             .asLink(controllers.routes.SummaryController.show(taxYear).url)
             .withId("continue")
         )
+        } else {
```

Comment:
We can parameterise the URL to eliminate the need for duplicating the button.

Reply:
I've implemented something in the controller now to replace this. Let me know what you think!